src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { partition } from "lodash"
 import * as React from "react"
 import * as Relay from "react-relay"
+import * as sharify from "sharify"
```

Comment:
Ok, I enabled the ‘disallow unused variables’ feature of the compiler and did a bug sweep.